libbeat/docs/communitybeats.asciidoc

```diff
@@ -92,6 +92,7 @@ https://github.com/radoondas/owmbeat[owmbeat]:: Open Weather Map beat to pull we
 https://github.com/joehillen/packagebeat[packagebeat]:: Collects information about system packages from package
 managers.
 https://github.com/WuerthIT/perfstatbeat[perfstatbeat]:: Collect performance metrics on the AIX operating system.
+https://github.com/stric-co/phishbeat[phishbeat]:: Lightweight beat for generating and monitoring for Phishing/Defamatory Domains.
```

Comment:
```suggestion
https://github.com/stric-co/phishbeat[phishbeat]:: Monitors Certificate Transparency logs for phishing and defamatory domains.
```

Reply:
For consistency, let's follow the convention of starting each description with a verb. 

IMO it's worth mentioning Certificate Transparency logs in this description. WDYT?